The version string isn't PEP 440 compatible, which means it won't work
with setuptools 66 or later.

--- a/setup.py	Fri Jan 19 06:40:42 2024
+++ b/setup.py	Sat Jul 20 19:07:05 2024
@@ -17,7 +17,7 @@
 
 setup(
       name='commix',
-      version='4.0-stable',
+      version='4.0+stable',
       description='Automated All-in-One OS Command Injection Exploitation Tool',
       long_description=open('README.md').read(),
       long_description_content_type='text/markdown',
